man/systemd.timer.xml

```diff
@@ -51,7 +51,9 @@
     but simply left running. There is no concept of spawning new service instances in this case. Due to this, services
     with <varname>RemainAfterExit=</varname> set (which stay around continuously even after the service's main process
     exited) are usually not suitable for activation via repetitive timers, as they will only be activated once, and
-    then stay around forever.</para>
+    then stay around forever. Targets, which by default do not deactivate on their own, can be activated repeatedly by
+    timers by setting <varname>StopWhenUnneeded=true</varname> on the target unit, so that it stops when all units
```

Comment:
Maybe break the sentence:
`Targets, which by default do not deactivate on their own, can be activated repeatedly by timers by setting <varname>StopWhenUnneeded=true</varname> on the target unit.`
`This will cause the target to be stopped when …`

Reply:
Split the sentence as suggested.